src/main/java/ru/bulldog/justmap/client/render/BufferedRenderer.java

```diff
@@ -109,7 +109,7 @@ protected void render(MatrixStack matrices, double scale) {
 		RenderSystem.setShaderColor(1.0F, 1.0F, 1.0F, 1.0F);
 		RenderSystem.setShader(GameRenderer::getPositionTexShader);
 		// yarn mapping missing for beginRead
```

Comment:
You can remove the comment now that it's not relevant

Reply:
Removed